### ISSUE TITLE
sssd_nss segfault detected during ipa-server-install.

### ISSUE THREAD

**Issue by pbrezina** [pbrezina/sssd-test#1406]
Cloned from Pagure issue: https://pagure.io/SSSD/sssd/issue/1402
- Created at 2012-07-04 10:02:58 by [shanks](https://pagure.io/user/shanks)
- Closed as Duplicate
- Assigned to nobody
---

Version: 
freeipa-server-2.99.0-0.20120702T2108Zgite5b6260.fc17.x86_64
sssd-1.8.94-0.20120702T2006Zgit5c9292b.fc17.x86_64

    Jul  4 03:48:11 dhcp201-193 systemd[1]: Reloading.
    Jul  4 03:48:12 dhcp201-193 sssd[15351]: Stopping sssd: [  OK  ]
    Jul  4 03:48:12 dhcp201-193 sssd: nscd socket was detected.  Nscd caching capabilities may conflict with SSSD for users and groups. It is recommended not to run nscd in parallel with SSSD, unless nscd is configured not to cache the passwd, group and netgroup nsswitch maps.
    Jul  4 03:48:12 dhcp201-193 sssd[15361]: Starting sssd: [  OK  ]#015[  OK  ]
    Jul  4 03:48:12 dhcp201-193 sssd: Starting up
    Jul  4 03:48:12 dhcp201-193 sssd[be[testrelm.com]]: Starting up
    Jul  4 03:48:12 dhcp201-193 sssd[nss]: Starting up
    Jul  4 03:48:12 dhcp201-193 kernel: [2237596.286717] sssd_nss[15373]: segfault at 10 ip 000000000044ee5b sp 00007fff15f0ec00 error 4 in sssd_nss[400000+7c000]
    Jul  4 03:48:12 dhcp201-193 sssd[pam]: Starting up
    Jul  4 03:48:12 dhcp201-193 kernel: [2237596.297271] sssd_pam[15374]: segfault at 10 ip 000000000044f31b sp 00007fff4d3c3780 error 4 in sssd_pam[400000+68000]
    Jul  4 03:48:12 dhcp201-193 systemd[1]: Reloading.
    Jul  4 03:48:12 dhcp201-193 abrt[15375]: Saved core dump of pid 15373 (/usr/libexec/sssd/sssd_nss) to /var/spool/abrt/ccpp-2012-07-04-03:48:12-15373 (1003520 bytes)
    Jul  4 03:48:12 dhcp201-193 abrt[15376]: Saved core dump of pid 15374 (/usr/libexec/sssd/sssd_pam) to /var/spool/abrt/ccpp-2012-07-04-03:48:12-15374 (1003520 bytes)
    Jul  4 03:48:12 dhcp201-193 abrtd: Directory 'ccpp-2012-07-04-03:48:12-15373' creation detected
    Jul  4 03:48:12 dhcp201-193 sssd[pam]: Starting up
    Jul  4 03:48:12 dhcp201-193 kernel: [2237596.432894] sssd_pam[15380]: segfault at 10 ip 000000000044f31b sp 00007fffb8b76670 error 4 in sssd_pam[400000+68000]
    Jul  4 03:48:12 dhcp201-193 sssd[nss]: Starting up
    Jul  4 03:48:12 dhcp201-193 kernel: [2237596.447245] sssd_nss[15385]: segfault at 10 ip 000000000044ee5b sp 00007fffb6dbcd20 error 4 in sssd_nss[400000+7c000]
    Jul  4 03:48:12 dhcp201-193 systemd[1]: Reloading.
    Jul  4 03:48:12 dhcp201-193 abrt[15386]: Saved core dump of pid 15380 (/usr/libexec/sssd/sssd_pam) to /var/spool/abrt/ccpp-2012-07-04-03:48:12-15380 (1003520 bytes)
    Jul  4 03:48:12 dhcp201-193 abrt[15388]: Saved core dump of pid 15385 (/usr/libexec/sssd/sssd_nss) to /var/spool/abrt/ccpp-2012-07-04-03:48:12-15385 (1003520 bytes)
    Jul  4 03:48:12 dhcp201-193 sssd[nss]: Starting up
    Jul  4 03:48:12 dhcp201-193 sssd[pam]: Starting up
    Jul  4 03:48:12 dhcp201-193 kernel: [2237596.548636] sssd_nss[15403]: segfault at 10 ip 000000000044ee5b sp 00007fff401bef00 error 4
    Jul  4 03:48:12 dhcp201-193 kernel: [2237596.554951] sssd_pam[15402]: segfault at 10 ip 000000000044f31b sp 00007fff79952e70 error 4 in sssd_pam[400000+68000]
    Jul  4 03:48:12 dhcp201-193 kernel: [2237596.557642]  in sssd_nss[400000+7c000]

### Comments

---

**Comment from [shanks](https://pagure.io/user/shanks) at [2012-07-04 10:03:23](https://pagure.io/SSSD/sssd/issue/1402#235421)**

backtrace
[bt.txt](/SSSD/sssd/issue/raw/files/c7d812bf83bb90ef1cf67f54de62b4c19dfdc2bda9b7c72c969133c3d1d538eb-bt.txt)

---

**Comment from [shanks](https://pagure.io/user/shanks) at [2012-07-04 13:01:54](https://pagure.io/SSSD/sssd/issue/1402#235422)**

No crash detected after commenting the local domain paramters in sssd.conf:

    [root@dhcp201-193 ~]# cat /etc/sssd/sssd.conf 
    [domain/testrelm.com]
    
    cache_credentials = True
    krb5_store_password_if_offline = True
    ipa_domain = testrelm.com
    id_provider = ipa
    auth_provider = ipa
    access_provider = ipa
    ipa_hostname = dhcp201-193.testrelm.com
    chpass_provider = ipa
    ipa_server = dhcp201-193.testrelm.com
    ldap_tls_cacert = /etc/ipa/ca.crt
    [sssd]
    config_file_version = 2
    #domains = LOCAL, testrelm.com
    domains = testrelm.com
    debug_level = 4
    services = nss, pam
    
    [nss]
    debug_level = 4
    filter_groups = root
    filter_users = root
    
    [pam]
    
    #[domain/LOCAL]
    #id_provider = local
    [root@dhcp201-193 ~]#

---

**Comment from [shanks](https://pagure.io/user/shanks) at [2012-07-05 15:23:50](https://pagure.io/SSSD/sssd/issue/1402#235423)**

Fields changed

milestone: NEEDS_TRIAGE => SSSD 1.9.0
priority: major => blocker

---

**Comment from [shanks](https://pagure.io/user/shanks) at [2012-07-12 18:57:36](https://pagure.io/SSSD/sssd/issue/1402#235424)**

Fields changed

rhbz:  => 0

---

**Comment from [shanks](https://pagure.io/user/shanks) at [2012-07-23 16:45:10](https://pagure.io/SSSD/sssd/issue/1402#235425)**

Nick reminded me that this is a duplicate of #1412. The backtrace and the symtopms are the same and I could no longer reproduce a segfault with LOCAL domain with the latest git HEAD.

Closing.

resolution:  => duplicate
status: new => closed

---

**Comment from [shanks](https://pagure.io/user/shanks) at [2017-02-24 14:51:56](https://pagure.io/SSSD/sssd/issue/1402#235426)**

**Metadata Update from @shanks**:
- Issue set to the milestone: SSSD 1.9.0
